packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -36,51 +38,64 @@ const MT5AccountCard = ({
+                >
+                    <div className='mt5-account-card__type--description'>
+                        <h1 className='mt5-account-card--paragraph mt5-account-card--paragraph-bold'>{title}</h1>
+                        {has_demo_banner && (
+                            <div className='mt5-account-card__banner mt5-account-card__banner--demo'>
+                                <Localize i18n_default_text='Demo' />
+                            </div>
+                        )}
+                        {existing_data && existing_data.display_balance && is_logged_in && (
+                            <p className='mt5-account-card--balance'>
+                                <Money
+                                    amount={existing_data.display_balance}
+                                    currency={existing_data.currency}
+                                    show_currency
+                                />
+                            </p>
+                        )}
+                        <p className='mt5-account-card--xsmall'>{descriptor}</p>
+                    </div>
+                    <div className='mt5-account-card__icon'>{icon && <IconComponent />}</div>
+                    <div className='mt5-account-card__background'>{background_image && <BackgroundComponent />}</div>
                 </div>
-            )}
-            <div
-                className={classNames('mt5-account-card__type', {
-                    'mt5-account-card__type--has-banner': has_popular_banner || has_demo_banner,
-                })}
-                id={`mt5_${type.category}_${type.type}`}
-            >
-                {icon && <IconComponent />}
-                <div className='mt5-account-card__type--description'>
-                    <h1 className='mt5-account-card--heading'>{title}</h1>
-                    {(!existing_data || !is_logged_in) && <p className='mt5-account-card--paragraph'>{descriptor}</p>}
-                    {existing_data && existing_data.display_balance && is_logged_in && (
-                        <p className='mt5-account-card--balance'>
-                            <Money
-                                amount={existing_data.display_balance}
-                                currency={existing_data.currency}
-                                show_currency
-                            />
-                        </p>
+                <div className='mt5-account-card__login-details'>
+                    {existing_data?.login && is_logged_in && <LoginBadge display_login={existing_data.display_login} />}
+                    {existing_data && is_logged_in && (
+                        <Button
+                            className='mt5-account-card__login-details-password'
+                            classNameSpan='mt5-account-card__login-details-password'
+                            onClick={() => {
+                                onPasswordManager(existing_data.login, title, type.category, type.type);
```

Comment:
`onPasswordManager` the name is a bit wierd :sweat_smile:

Reply:
Hahaha, it is also part of other people's code so it is better untouched 😄